samples/nodejs-compat-streams-split2/README.md

```diff
@@ -0,0 +1,28 @@
+# Node.js Compat Example
+
+To run the example on http://localhost:8080
+
+```sh
+$ ./workerd serve config.capnp
+```
+
+To run using bazel
+
+```sh
+$ bazel run //src/workerd/server:workerd -- serve ~/cloudflare/workerd/samples/nodejs-compat/config.capnp
```

Comment:
Cut-and-past-o here with `nodejs-compat` in the path:

`~/cloudflare/workerd/samples/nodejs-compat/config.capnp`

Suggest `$(realpath samples/nodejs-compat-streams-split2/config.capnp)`